### PR TITLE
(Sentinel) Fix wallet string not being marshalled from por

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

Please include a summary of the changes and the related issue.
Please also include relevant motivation and context.
List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

**Comment by nick-bisonai** [comment]
@coderabbitai review

File: sentinel/pkg/checker/balance/app.go [hunk 1]
```diff
@@ -249,15 +254,30 @@ func loadWalletFromOraklAdmin(ctx context.Context, url string) ([]Wallet, error)
...
 	if err != nil {
 		return wallet, err
 	}
-	address := common.HexToAddress(reporter)
+
+	if resp.StatusCode != http.StatusOK {
+		log.Info().
+			Int("status", resp.StatusCode).
+			Str("url", url+porEndpoint).
+			Msg("failed to make request")
```
Comment (Intizar-T) [inline sentinel/pkg/checker/balance/app.go:266]: just a minor suggestion, how about more detailed error log? such as the file location, which function, etc. Or will that kind of info automatically show up in the logs?

**Review by Intizar-T — APPROVED** [review]
lgtm!

File: sentinel/pkg/checker/balance/app.go (hunk 1 above)
Comment (nick-bisonai) [reply sentinel/pkg/checker/balance/app.go:266]: good point, will make an update

**Pull request merged**